### DISCUSSION TITLE
Ipad Out of Sync without syncing a byte

### DISCUSSION THREAD

**Discussion by lqzab** [MobiusSync/MobiusSync#79] [General]
A folder which I share from my NAS and my phone, containing films, does not sync at all. For a second preparing sync flashes, but for only less than a second. In the shared folder there are 3 films (2 times 2GB and one 600MB). I don't get any other error message. Please help
![3BA833DC-99C9-49E0-AF38-EF3E1B5078D9](https://user-images.githubusercontent.com/106881405/172016939-7d6957dd-7d58-4f17-9ac9-561966bb2f21.jpeg)

**Comment by SimonPickup** [comment]
Did you get this sorted? Have you upgraded for unlimited file sync, as the folder size is well above the limit.
